scipy/_lib/_testutils.py

```diff
@@ -94,18 +94,6 @@ def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
 
         pytest_args = ['--showlocals', '--tb=short']
 
-        if doctests:
```

Comment:
One place less to look for skips is nice. It's now `conftest.py` (docs) or `pytest.ini` (regular tests)